docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -604,27 +621,68 @@ guess of what specific sex- and gender-related factors could be associated with
 
   Reference 'measurement error' section again here.
 
-4.2 Further Results (Text, Figures, Tables)
--------------------------------------------
+4.4 Result indicator definition
+-------------------------------
 
-Of course, it is not possible to include a long explanation each and every time sex- or
-gender-related concepts are referenced.
-After an initial, comprehensive description, we have to switch to some kind of shorthand.
+In *each* stand-alone document (paper, table, figure, etc) where we present results
+that relate to sex and/or gender,
+we should define our result indicator (or indicators, though we
+usually won't be able to report more than one).
 
-The most important things to keep in mind when choosing this shorthand are:
+How an indicator should be defined depends on the data that informed it.
+In general, we think of an indicator as the "least common denominator"
+of the data that went into it.
 
-- If the full term includes both sex and gender, both should appear in it in some way
-  -- using only one or the other invites the reader to jump to conclusions.
-- Likewise, we should not use the basic terms "males" and "females" or "men" and "women"
-  when we are reporting a concept that is more complex than correctly-measured sex or gender alone.
-- If the full term includes an acknowledgement that results are based on proxy measures,
-  that should be preserved, or at least nodded to, in the shorthand.
+Put another way, when the sex/gender-related factors we are reporting on are unclear - 
+for example when we have assumed equivalence between different measures across 
+data sources - we should use terms that reflect this uncertainty.
+
+To understand this, we can look at our figure of hypothetical data sources
+from above. There is now an additional row with how we would include
+this data in our analysis. You can see the first two data sources would both be
+undifferentiated sex/gender. This is due to ambiguity between sex and gender in
+the question asked.
+
+The latter two data sources more clearly reflect sex, although one is sex assigned
+at birth and the other is sex identity. Therefore if you included only data sources
```

Comment:
```suggestion
at birth and the other is gender identity. Therefore if you included only data sources
```
?

Reply:
No, I set up this figure to ask explicitly about what is your sex, partially to set up a section saying that these data sources both clearly reference sex.

Reply:
Yes, agree that it should be "sex" and not "gender" (note that I didn't edit this in this PR, it is just moved around). On a cursory look at our cited survey methodology sources, I actually can't find a reference for this meaning of the term "sex identity" -- perhaps something like "self-reported current sex" could be less ambiguous. See for example [the "Allowable alternative question module" for sex from the Australian Bureau of Statistics.](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#sex)